build_debian.sh

```diff
@@ -266,6 +266,10 @@ sudo mv $FILESYSTEM_ROOT/grub-pc-bin*.deb $FILESYSTEM_ROOT/$PLATFORM_DIR/x86_64-
 ## Disable kexec supported reboot which was installed by default
 sudo sed -i 's/LOAD_KEXEC=true/LOAD_KEXEC=false/' $FILESYSTEM_ROOT/etc/default/kexec
 
+## Modifty ntp default configuration: disable initial jump (add -x), and disable
+## jump when time difference is greater than 1000 seconds (remove -g).
+sudo sed -i "s/NTPD_OPTS='-g'/NTPD_OPTS='-x'/" $FILESYSTEM_ROOT/etc/default/ntp
```

Comment:
not particular this pr, I am wondering if we need to check the return code, if '-g' is not matched due to configuration change, we might now have '-x'. this is true for other sed,  i think we need to flag this in the build time.

Reply:
The build will fail if the match fails. I found that while working out the correct regex.